actions/common.go

```diff
@@ -1,20 +1,43 @@
 package actions
 
 var (
-	repoNames = []string{
-		"builder",
-		"controller",
-		"dockerbuilder",
-		"fluentd",
-		"logger",
-		"minio",
-		"postgres",
-		"registry",
-		"router",
-		"slugbuilder",
-		"slugrunner",
-		"workflow",
-		"workflow-e2e",
-		"workflow-manager",
+	// TODO: pull repo list from canonical source -- perhaps workflow-manager-api?
+	repoToComponentNames = map[string][]string{
+		"builder":          {"Builder"},
+		"controller":       {"Controller"},
+		"dockerbuilder":    {"DockerBuilder"},
+		"fluentd":          {"FluentD"},
+		"monitor":          {"InfluxDB", "Grafana", "Telegraf"},
+		"logger":           {"Logger"},
+		"minio":            {"Minio"},
+		"postgres":         {"Database"},
+		"registry":         {"Registry"},
+		"router":           {"Router"},
+		"slugbuilder":      {"SlugBuilder"},
+		"slugrunner":       {"SlugRunner"},
+		"workflow":         {"Workflow"},
+		"workflow-e2e":     {"WorkflowE2E"},
+		"workflow-manager": {"WorkflowManager"},
 	}
+
+	repoNames      = getRepoNames(repoToComponentNames)
+	componentNames = getComponentNames(repoToComponentNames)
 )
+
+func getRepoNames(repoToComponentNames map[string][]string) []string {
+	repoNames := make([]string, 0, len(repoToComponentNames))
```

Comment:
Including the `0` seems to prevent an empty entry in the `repoNames` slice, therefore preventing a GitHub API call down the line with an empty repo name.  Does that make sense?  Is there a different spot I should look at for preventing said issue?